src/core/server/elasticsearch/elasticsearch_service.mock.ts

```diff
@@ -55,14 +89,16 @@ const createMock = () => {
     start: jest.fn(),
     stop: jest.fn(),
   };
-  mocked.setup.mockResolvedValue(createSetupContractMock());
+  mocked.setup.mockResolvedValue(createInternalSetupContractMock());
   mocked.stop.mockResolvedValue();
   return mocked;
 };
 
 export const elasticsearchServiceMock = {
   create: createMock,
-  createSetupContract: createSetupContractMock,
+  createInternalSetup: createInternalSetupContractMock,
+  createSetup: createSetupContractMock,
```

Comment:
I think I'm in favor of this 'new' naming without the `contract` suffix, however this is inconsistent with every other service mocks we got. Should we decide to go with this and change the others when we can?

Reply:
I think we should unify this naming. Right now we have inconsistency across the codebase. [example](https://github.com/restrry/kibana/blob/2c8513af235e905231421c21601c98a46d41b1e7/src/core/server/http/http_service.mock.ts#L110)
Should we add this to NP conventions and fix all the mocks?

Reply:
This is minor, but I think we should

Reply:
created https://github.com/elastic/kibana/issues/54229